_integration-schemas/zendesk/organizations.md

```diff
@@ -103,4 +103,4 @@ attributes:
   - name: "url"
     type: "string"
     description: "The API URL of the organization."
----
\ No newline at end of file
+---
```

Comment:
I'm not sure what's up with line 106. Is that expected?

Reply:
@dylan-stitch You mean just the three dashes? Yes. It's in the Metadocs as well, but [this is the official Jekyll documentation explaining it](https://jekyllrb.com/docs/front-matter/).

Basically, these dashes need to be present for Jekyll to detect these files. Without them, Jekyll won't throw an error - they'll just be ignored and therefore not rendered.